aurget

```diff
@@ -189,20 +192,20 @@ enter_build_directory() {
 }
 
 buildpkg() {
-  debug "executing $colorY$makepkg $makepkg_options $*$nocolor"
+  debug "executing $colorY$makepkg ${makepkg_options[*]} $*$nocolor"
 
   if $silent; then
     debug 'building silently...'
-    $makepkg $makepkg_options "$@" --noconfirm &>/dev/null
+    $makepkg "${makepkg_options[@]}" "$@" --noconfirm &>/dev/null
```

Comment:
I'm surprised ShellCheck doesn't complain about the un-quoted `$makepkg`, do you know why?

Reply:
I was surprised as well, that `$makepkg` does not need quotes (I initially thought that the error is only shown if the one with the `$makepkg_options` issue is resolved.

Should I add the quotes? Should I do that in another commit, such that it might be easier to revert this change in future? But I cannot find anything regarding this, nor do I know of any problems.

Reply:
Mh I looked into that, I think we should quote it.

The only disadvantage would be that it is not possible to put options, e.g. one cannot set `makepkg -d` as default, but that is something one probably do not want to do, since the `-d` belongs into the options. I cannot find any other difference.